src/mcp_jenkins/server/build.py

```diff
@@ -101,3 +101,91 @@ async def stop_build(ctx: Context, fullname: str, build_number: int) -> None:
         build_number: The number of the build to stop
     """
     return client(ctx).build.stop_build(fullname, build_number)
+
+
+@mcp.tool(tag='read')
+async def get_test_results(
+    ctx: Context,
+    fullname: str,
+    build_number: int | None = None,
+    status_filter: list[Literal['PASSED', 'SKIPPED', 'FAILED', 'FIXED', 'REGRESSION']] | None = None
```

Comment:
The filtering is important to reduce the context returned when a job can have a large number of tests, when only a couple failed.